release/RELEASE.md

```diff
@@ -31,8 +31,12 @@ patches we backport to the 1.0 release branch).
    $ ./release.sh snapshot cc880e2 0.1.2
    cc880e290b2311d0bf05d58c7d75c50784c0131c 0.1.2-snapshot.20200513.4174.0.cc880e29
    ```
-   Then open a PR with the changed `LATEST` file, and add the `Standard-Change`
-   label.
+   Then open a PR _to be merged in `main`_ (even if it's for a maintenance release)
+   with the changed `LATEST` file, add the line produced by the `release.sh`
+   invocation in a meaningful position and add the `Standard-Change` label. It
```

Comment:
Please elaborate here what 'meaningful position` means, i.e., you should keep the list of versions sorted from newest to oldest.

Reply:
I actually left it vague on purpose (and I am appealing to common sense) because version order and time order can be (and are now already) different and mandating one over the other can be confusing IMO, also considering that code changes are not linear across branches.

@garyverhaegen-da @moritzkiefer-da WDYT?

Reply:
in principle semver ordering would be the right thing to do but that falls apart once we have weird things like the 1.8 snapshot so maybe state something like "if you’re not sure, semver ordering is probably the right thing to do".

Reply:
I agree on semver ordering. Everything should order nicely by semver, as that's why I designed the version numbers that way. Unless we somehow gave a random version number to an official release at some point 🤷

Reply:
Ah right I see what you mean. `1.8.0-snapshot.20201214.5841.0.a8ae8e4a` is garbage.  Not sure how to handle that; I'd go for renaming it to 1.8.1 and renaming the current breed of new-1.8 1.8.2. I don't think it's a big issue that 1.8.2 is not strictly built on top of 1.8.1. Another option would be to call it something like `1.8.0-garbage`, but I'm not 100% confident we don't have some code somewhere that expects exactly `-snapshot`, so I'd have to check for that first.